app/src/main/res/layout/fragment_send_messages.xml

```diff
@@ -0,0 +1,52 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.teamagam.gimelgimel.app.message.viewModel.SendMessagesViewModel" />
+    </data>
+
+    <LinearLayout
+        android:id="@+id/fragment_messages_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:orientation="horizontal">
+
+        <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="@dimen/fab_normal"
```

Comment:
This dimension needs to be defined at values/dimens.xml too
Currently it's defined only on w820dp\dimens.xml.